packages/renderer/src/lib/ImagesList.svelte

```diff
@@ -99,11 +102,12 @@ function getId(imageInfo: ImageInfo) {
 }
 
 function getTag(imageInfo: ImageInfo) {
-  // get name
+  if(!imageInfo.RepoTags){
+    return '<node>'
```

Comment:
@gorkem @benoitf shouldn't this be `<none>`?

Reply:
fixed by https://github.com/containers/desktop/commit/676eda75e9fc47ad21d7f3194f73cf84fb0894bf